src/module/freebsd/freebsd.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+// FreeBSD specific definitions
+#include "syscalls.h"
+
+#define SAFE_WRITE(x) x // ???
+#define LOG(...) uprintf(__VA_ARGS__)
+#define SYSCALL_TABLE sysent
+#define TARGETING_PROTO struct thread *td
+#define TARGETING_PARMS td
+#define PERSONALITY() (td->td_proc->p_flag2)
```

Comment:
Let's prefix these macros with `KRF_`, just to avoid accidentally clobbering any macros/tests in the Linux and FreeBSD headers. e.g., `KRF_PERSONALITY`.